contrib/skopeo/patches/unphony-bin-skopeo.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/Makefile b/Makefile
```

Comment:
this patch doesn't do anything

Reply:
It does. It prevents the `bin/skopeo` from being marked `.PHONY` which causes it to rebuild on install actions.

I should have added a comment on top of the patch though!

Reply:
it does not cause it to be rebuilt at all though; the patch does not do anything.

you only call install-docs and install-completions, none of which depend on bin/skopeo, so this doesn't do anything. you are wrong.

Reply:
technically it gets called with the addition of the tests below, but before that this truly does nothing at all

Reply:
> it does not cause it to be rebuilt at all though; the patch does not do anything.
> 
> you only call install-docs and install-completions, none of which depend on bin/skopeo, so this doesn't do anything. you are wrong.

I am not wrong. Please don't accuse others of being wrong without checking. `install-completions` depend on `completions` which depend on `bin/skopeo`....

Reply:
conveniently, this whole time i was reading 1.6.0

Reply:
It was the case with 1.6.0 already. 1.16.0 only allowed me to drop the device mapper dependency.

Reply:
Ah, I see what you mean with 1.6.0. Indeed, they did not use that with completions.

Reply:
Unlucky typo for all this confusion. :S

Reply:
in 1.6.0 there is no completions: target and install-completions/install-docs definitely don't depend on this https://github.com/containers/skopeo/blob/e4b79d7741c40434838ece2790be537083fc2a04/Makefile#L169

i admit i'm completely wrong for 1.16 (so all the changes here are fine; don't worry about it), but on 1.6 this was incorrect

Reply:
>Unlucky typo for all this confusion. :S

it's my bad too, i should've double checked..

sorry :/

Reply:
> > Unlucky typo for all this confusion. :S
> 
> it's my bad too, i should've double checked..
> 
> sorry :/

No worries. You've been very diligent with the review!